src/styles/feedback.css

```diff
@@ -0,0 +1,31 @@
+.feedback {
+  display: flex;
+  align-items: flex-end;
```

Comment:
`align-items: flex-end;` doesn't seem to impact the layout - remove if true.

also your tests seem to be weirdly offset - unlike the deployed page
please try running tests again. If pass - do another `npm run deploy` so that latest test results are in place.